ansible.cfg

```diff
@@ -8,6 +8,15 @@ roles_path = ./roles
 # Be sure the user running Ansible has permissions on the logfile
 log_path = /var/log/ansible.log
 
+forks = 20 
+host_key_checking = False
+gathering = smart
+fact_caching = jsonfile
+fact_caching_connection = $HOME/ansible/facts
```

Comment:
Will ansible create this directory?

Reply:
Yes, Ansible creates it the first time it gathers facts.